### ISSUE TITLE
Installing Two Apps after Gradle sync

### ISSUE THREAD

**Issue by rohitsangwan01** [daandtu/android-web-scraper#2]
After implementing in gradle , android studio instalin two apk, one that i worked on,and another of Google webview and i have just one launcher activity

**Comment by ajaykumarpprajapati** [comment]
Installing two apk on device.

**Comment by rohitsangwan01** [comment]
Yes , One Apk is Opening Webview ,and Another is One I worked on

**Comment by venky9885** [comment]
yes same issue
